### DISCUSSION TITLE
Is v0.5.0 viable for use?

### DISCUSSION THREAD

**Discussion by lucasogit** [wehr-lab/autopilot#207] [General]
Title. Or is it better to stick with v0.4.4.post0 for now? We were wanting to swap to v0.5, but I want your opinion @sneakers-the-rat if you thought it would be viable for us to upgrade.

**Comment by sneakers-the-rat** [comment]
I would not upgrade right now! still haven't figured out the PySide6 problems yet. That was the only real change, because I was trying to leave it with versions of dependencies that weren't EOL, but every Qt upgrade is its own special hell.

I'm in a workshop this week and will be able to get to it next week, keep pinging me.

**Reply by sneakers-the-rat** [reply]
It's likely going to be a pretty perplexing bug. I might just take the route of rolling back to PySide2

**Reply by lucasogit** [reply]
What version would you recommend we use in the meantime? I see besides main there is a v0.5.0, v0.4.5, and v0.4.4.post0. I think we are currently on v0.4.4.post0.

**Reply by lucasogit** [reply]
bump

**Reply by sneakers-the-rat** [reply]
holy hell when did it become august.

let's find some time to talk about this, i was supposed to do that ages ago with you and mike. are you free tomorrow?

**Reply by lucasogit** [reply]
time flies when you're having fun. i'll be around tomorrow after like 12pm or so. want to just give me a call when you get the chance (or whatever form of communication works best for you)?

**Reply by lucasogit** [reply]
> What version would you recommend we use in the meantime? I see besides main there is a v0.5.0, v0.4.5, and v0.4.4.post0. I think we are currently on v0.4.4.post0.

just here to bump this again
